src/utils/Database/LSMTree/Util/Orphans.hs

```diff
@@ -44,31 +45,34 @@ deriving anyclass instance Uniform Word256
 instance Hashable Word256 where
   hashIO32 (Word256 a b c d) = hashIO32 (a, b, c, d)
 
-instance Serialise Word256 where
-  serialise (Word256{word256hi, word256m1, word256m0, word256lo}) =
+instance SerialiseKey Word256 where
+  serialiseKey (Word256{word256hi, word256m1, word256m0, word256lo}) =
       fromByteString $ B.toLazyByteString $ mconcat [
           B.word64BE word256hi
         , B.word64BE word256m1
         , B.word64BE word256m0
         , B.word64BE word256lo
         ]
     where
-      fromByteString :: LBS.ByteString -> SerialisedKey
+      fromByteString :: LBS.ByteString -> RawBytes
       fromByteString =
             fromShortByteString
           . SBS.toShort
           . LBS.toStrict
+  deserialiseKeyOrFail = error "deserialiseKeyOrFail: Word256" -- TODO
```

Comment:
Are you still planning to add the implementation within this PR?

Reply:
Wasn't planning on it, since it's only a utility and we're not using deserialisation yet